src/containers/Dev/Snapshots.tsx

```diff
@@ -64,7 +64,11 @@ const Snapshots: React.FC = () => {
         url: getServerUrl('snapshot', { action: 'use', name }),
       })
 
-      if (resultJson.success) return setIsLoading(false)
+      if (resultJson.success) {
+        setIsLoading(false)
+        location.reload(true)
+        return
+      }
```

Comment:
Not related to this PR. This just makes the whole app reload after a snapshot is loaded, which is much nicer.

Reply:
Oh! Cool, I'll have a look on how this is looking : )